src/figuresClasses.ts

```diff
@@ -1,19 +1,88 @@
+
 export interface Figure {
+  shape: string,
+  color: string,
+  getArea(): number,
+}
+
+type Color = 'red' | 'green' | 'blue';
 
+enum Shape {
```

Comment:
you can do it, but in my humble opinion, 
there should be 2 TYPES or 2 ENUMS =)


btw, it's correct, so nevermind